package.json

```diff
@@ -27,6 +27,7 @@
   "dependencies": {
     "lodash.assign": "^4.2.0",
     "lodash.intersection": "^4.4.0",
+    "lodash.isplainobject": "^4.0.6",
```

Comment:
Btw the more functional nature of LiveScript makes it more idiomatic to use http://www.preludels.com instead of lodash. This allows for way more clean and elegant code. Prelude is pretty feature-complete, and kind of a lodash for LiveScript. You should at least check it out. A big part of the beauty of LiveScript comes from being able to use that library in cool ways.